sprockets/mixins/metrics/statsd.py

```diff
@@ -191,5 +230,14 @@ def install(application, **kwargs):
     if 'port' not in kwargs:
         kwargs['port'] = os.environ.get('STATSD_PORT', '8125')
 
+    if 'proto' not in kwargs:
+        if "STATSD_PROTO" in os.environ:
```

Comment:
Don't abbreviate here to save 3 characters. Either do something like `STATSD_TCP` or use `STATSD_PROTOCOL`